### PR TITLE
Support audio codecs in video

### LINKED ISSUE #1430: Standard recommends using <video> element for audio to allow accessibility, but does not mention how this impacts/is impacted by format support between the two media elements in a UA
In https://w3c.github.io/html/semantics-embedded-content.html#the-audio-element, there is a NOTE:

> In particular, this content is not intended to address accessibility concerns. To make audio content accessible to the deaf or to those with other physical or cognitive disabilities, a variety of features are available. If captions or a sign language video are available, the video element can be used instead of the audio element to play the audio, allowing users to enable the visual alternatives.

Later, in https://w3c.github.io/html/semantics-embedded-content.html#the-media-elements, there is a NOTE:

> Both audio and video elements can be used for both audio and video. The main difference between the two is simply that the audio element has no playback area for visual content (such as video or captions), whereas the video element does.

But there is no explicit requirement that a UA supports the same mux and codec formats in a `<video>` tag as it does in an `<audio>` tag. Therefore this method of making an audio track accessible to deaf users by using a `<video>` tag instead of an `<audio>` tag and adding a text `<track>` may be unsupported.

This issue is related to #1429, where site designers currently may incorrectly include a subtitles track in an `<audio>` element in the belief that it makes the audio track "fully accessible".

### PR DESCRIPTION
**Pull request by chaals**
fix #1430 

Note that user agents should support playback of all supported audio codecs in video element for accessibility.

### CODE REVIEW

**Review by scottaohara — APPROVED** [review]

**Comment by scottaohara** [comment]
Marked as non substantive for IPR from ash-nazg.

**Comment by chaals** [comment]
@scottaohara I am assuming you will merge this when you think it is ready, or request changes.

File: sections/semantics-embedded-content.include [hunk 1]
```diff
@@ -5676,6 +5676,10 @@ attribute's value is a type that a <a>plugin</a> supports, then the value of the
   The <{video}> element is a <a>media element</a> whose <a>media data</a> is ostensibly video
   data, possibly with associated audio data.
 
+  Because the <{video}> element is necessary to support accessibility for audio content,
+  user agents should support playback of the same set of audio formats in the <{video}> element
```
Comment (OwenEdwards) [inline sections/semantics-embedded-content.include:5680]: Should this reference "audio formats **and container formats** in the <{video}> element"?

Comment (chaals) [reply sections/semantics-embedded-content.include:5680]: Yes, that seems reasonable.

**Pull request merged**